### PR TITLE
Connect react redux

### PR DESCRIPTION
**Pull request by astroboyReloaded**
- Create an `initialState `variable for your `booksSlice`.
- Display your `books`, received from the slice, in a reusable component.
- Dispatch actions using `useDispatch`.
- Add a `<Button>` component, which includes:
   - An event handler that **adds** a book to the `books` array (with attributes id, title, and author).
- Add a `<Button>` component, which includes:
   - An event handler that **removes** a book from the `books` array (by id).

### CODE REVIEW

File: src/components/AddNewBook.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    }));
+    title.current.value = '';
+    author.current.value = '';
+    title.current.focus();
+  };
+
+  return (
+    <section>
+      <h2>ADD NEW BOOK</h2>
+      <form onSubmit={handleAddBook}>
```
Comment (Shedrack-Sunday) [inline src/components/AddNewBook.js]: - [ ] Kindly implement this handler in the  `<Button>` but in the button case, it should include an event handler to add a book to the books array (with attributes title and author)

### Sample:
```
<Button onClick={() => dispatch(addBook({ title, author }))}>
        Add book
      </Button>
```

## Requirements:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @astroboyReloaded  👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Nice work with the use of React libraries ✔️ 
- Good project setup ✔️ 
  
# Required Changes  🌵 

- [ ] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you make In line with the requirement


_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/AddNewBook.js (hunk 1 above)
Comment (astroboyReloaded) [reply src/components/AddNewBook.js]: Thank you very much for your comments, @Shedrack-Sunday!
I do have a question, does it have to have a `type="submit"`, or can it be a `type="button"`?

Comment (Shedrack-Sunday) [reply src/components/AddNewBook.js]: @astroboyReloaded, You have a similar implementation the `Remove Button`. I'll advise you to follow the same pattern. It's going to be a button component with an event handler.

**Review by uwadonat — APPROVED** [review]
## Hi @astroboyReloaded,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉
### Highlights
- Well-organized and clean code ✔️
- Detailed pull request message ✔️
approv

# Status: Approved ✔️✔️✔️

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**